app/controllers/alchemy/json_api/pages_controller.rb

```diff
@@ -36,6 +35,17 @@ def show
 
       private
 
+      def render_pages_json(allowed)
+        # Only load pages with all includes when browser cache is stale
```

Comment:
The comment here should probably be in the calling code

Reply:
In both calling places? Na..